deploy/BUILD.bazel

```diff
@@ -34,6 +38,40 @@ alias(
     actual = "//driver/src/main/java/com/code_intelligence/jazzer/junit:junit.publish",
 )
 
+sh_test(
+    name = "jazzer-api_artifact_test",
+    srcs = ["jazzer-api_artifact_test.sh"],
```

Comment:
I wonder if we should move the `junit` artifact and `junit_artifact_test` also into `deploy` to keep exports more closely together.

Reply:
Good idea, done. Got quite some code savings that way.